Weather/WeatherWidget.swift

```diff
@@ -9,66 +9,151 @@
 import Foundation
 import AppKit
 import PockKit
-
+public var Info = 0
+public var Day = 0
+public var DoubleTap = 0
+public var Wait = 0
 class WeatherView: PKDetailView {
     override func didLoad() {
         canScrollTitle = true
-		canScrollSubtitle = true
-		set(title: "Weather")
-		set(subtitle: "Fetching data")
-		set(image: NSImage(named: NSImage.touchBarSearchTemplateName))
+        canScrollSubtitle = true
+        set(title: "Weather")
+        set(subtitle: "Fetching data")
+        set(image: NSImage(named: NSImage.touchBarSearchTemplateName))
         super.didLoad()
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+            NotificationCenter.default.post(name: .didChangeWidgetPreferences, object: nil)
+        }
     }
     override func didTapHandler() {
```

Comment:
Double tap, triple tap and single tap detection. Opens weather app when taped once. In the future double and triple tap will show info like UV Index, Feels Like and more.